paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -179,6 +180,68 @@ class PaymentSheet internal constructor(
         IntentConfirmationInterceptor.createIntentCallback = createIntentCallback
     }
 
+    /**
+     * Builder to add optional callbacks to [PaymentSheet].
+     *
+     * @param paymentResultCallback Called with the result of the payment after [PaymentSheet] is dismissed.
+     */
+    class Builder(internal val resultCallback: PaymentSheetResultCallback) {
+        internal var externalPaymentMethodConfirmHandler: ExternalPaymentMethodConfirmHandler? = null
+            private set
+        internal var createIntentCallback: CreateIntentCallback? = null
+            private set
+
+        /**
+         * @param handler Called when a user confirms payment for an external payment method.
+         */
+        fun externalPaymentMethodConfirmHandler(handler: ExternalPaymentMethodConfirmHandler) = apply {
+            this.externalPaymentMethodConfirmHandler = handler
+        }
+
+        /**
+         * @param createIntentCallback Called when the customer confirms the payment or setup.
+         */
+        fun createIntentCallback(createIntentCallback: CreateIntentCallback) = apply {
+            this.createIntentCallback = createIntentCallback
+        }
+
+        /**
+         * Returns a [PaymentSheet] and initializes callback handlers if respective callbacks are set
+         *
+         * @param activity The Activity that is presenting [PaymentSheet].
+         */
+        fun build(activity: ComponentActivity): PaymentSheet {
+            initializeCallbacks()
+            return PaymentSheet(DefaultPaymentSheetLauncher(activity, this.resultCallback))
+        }
+
+        /**
+         * Returns a [PaymentSheet] and initializes callback handlers if respective callbacks are set
+         *
+         * @param fragment the Fragment that is presenting the payment sheet.
+         */
+        fun build(fragment: Fragment): PaymentSheet {
+            initializeCallbacks()
+            return PaymentSheet(DefaultPaymentSheetLauncher(fragment, this.resultCallback))
```

Comment:
```suggestion
            return PaymentSheet(DefaultPaymentSheetLauncher(fragment,resultCallback))
```

I think there's a lot of other similar unnecessary `this.`

Reply:
Removed 👍